test/controllers/observer_controller_test.rb

```diff
@@ -611,6 +623,29 @@ def test_map_observations
     assert_template(:map_observations)
   end
 
+  def test_map_observation_hidden_gps
+    obs = observations(:unknown_with_lat_long)
+    get(:map_observation, params: { id: obs.id })
+    assert_true(assigns(:observations).map(&:lat).map(&:to_s).join("").include?("34.1622"))
```

Comment:
```suggestion
    assert_false(assigns(:observations).map(&:lat).map(&:to_s).join("").
                                        include?("34.1622"))
```
At the moment, we're having RuboCop mask line length offenses (max 80) because there are so many.
But one day, I'll have to fix them. So best not to add another